validate-artifacts.ps1

```diff
@@ -0,0 +1,39 @@
+
+# default script values 
+$artifactsConfigPath = "$PWD/artifacts.json"
+$resourcesTemplateName = "exported-template.json"
+$taskName = "task1"
+$tempFolderPath = "$PWD/temp"
+
+Write-Output "Reading config" 
+$artifactsConfig = Get-Content -Path $artifactsConfigPath | ConvertFrom-Json 
+
+Write-Output "Checking if temp folder exists"
+if (-not (Test-Path "$tempFolderPath")) { 
+    Write-Output "Temp folder does not exist, creating..."
+    New-Item -ItemType Directory -Path $tempFolderPath
+}
+
+Write-Output "Downloading artifacts"
+
+if (-not $artifactsConfig.resourcesTemplate) { 
+    Write-Error "Artifact config value 'resourcesTemplate' is empty! Please make sure that you executed the script 'scripts/generate-artifacts.ps1', and commited your changes"
+} 
+Invoke-WebRequest -Uri $artifactsConfig.resourcesTemplate -OutFile "$tempFolderPath/$resourcesTemplateName" -UseBasicParsing
```

Comment:
The `Invoke-WebRequest` cmdlet is used here to download the resource template. Ensure that the URL in `$artifactsConfig.resourcesTemplate` is accessible and correct. If there are any issues with the URL, the download will fail.